content/command-line/concepts/bash/terms/input-output-redirection/input-output-redirection.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'Bash: Input-Output Redirection'
+Description: 'Redirection allows the modification of the destination of standard input and standard output in a shell.'
```

Comment:
This can be written in a more direct and simplified way - 

```suggestion
Description: 'Redirection modifies the destination of standard input and output in a shell.'
```